### ISSUE TITLE
Fix dash saving state error

### ISSUE THREAD

**Issue by naterush** [mito-ds/mito#973]
**Describe the bug**

This is a result of our hidden global variables. We should figure out if we can get rid of this...

**Comment by DarkCTO** [comment]
Yes it is affecting dash applications, between Dash sessions, even when the ids are all different, they keep pulling dataframes from previous sessions, and cannot be cleared until server is shut down and back up.

**Comment by naterush** [comment]
@DarkCTO thanks for the comment here! We've made some progress on this in this PR: https://github.com/mito-ds/mito/pull/1009

**This should be deployed by the end of the week, and the docs will be updated. This should fix this issue for now -- so checkin at the end of the week.**

**Comment by aarondr77** [comment]
@DarkCTO I just released a new version of the Spreadsheet component that should now work across multiple users. 

You can see an example of the new API in use [here](https://docs.trymito.io/mito-for-dash/api-reference#the-spreadsheet-component). 

Note that there are two important changes: 
1. The addition of the `activate_mito` function that setups communication between the Mito frontend and the backend
2. The updated `id` with the format `{type: "spreadsheet", id: str}` that allows users to have separate Spreadsheet components to edit.

**Comment by DarkCTO** [comment]
Great addition, I'll be trying this.
Requiring id to be of keys type and id is less flexible than allowing dash's pattern-matching IDs
https://dash.plotly.com/pattern-matching-callbacks
as I have uuid assigned to each components

But a great step toward stateless mitosheet!

**Comment by DarkCTO** [comment]
Yep, works well.
Looking at the commit, I see it was kinda patchy. But it works for now!
Appreciate your work !
